exporting-grafana-snapshots.md

```diff
@@ -5,7 +5,7 @@ summary: Learn how to export snapshots of Grafana Dashboard, and how to visualiz
 
 > **Warning:**
 >
-> - Since TiDB v6.0.0, PingCAP no longer maintains MetricsTool.
+> - Since TiDB v6.0.0, PingCAP no longer maintains MetricsTool. Since v6.1.0, PingCAP no longer maintains the MetricsTool documentation.
```

Comment:
There is only one document that introduces MetricsTool, which is this document.
```suggestion
> - Since TiDB v6.0.0, PingCAP no longer maintains MetricsTool. Since v6.1.0, PingCAP no longer maintains the MetricsTool document.
```